.github/workflows/release-prod.yml

```diff
@@ -39,8 +40,6 @@ jobs:
         id: changesets
         uses: changesets/action@v1
         with:
-          title: Version Packages
-          commit: "chore(repo): Version packages"
```

Comment:
❓ @LekoArts  why did we remove this?

Reply:
With this the PRs/Commits will have the title "Version Packages (alpha-v5)" instead of "Version Packages" -- and we don't need to change it for every change of pre tag or new major